app/services/taxonomy/taxon_unpublisher.rb

```diff
@@ -0,0 +1,41 @@
+module Taxonomy
+  class TaxonUnpublisher
+    def self.call(taxon_content_id:, redirect_to_content_id:, user:, retag: true)
+      new.unpublish(taxon_content_id: taxon_content_id,
```

Comment:
Will the "tag to parent" checkbox be selected by default in the form? If it's not, it's feels odd that the default behaviour is true.

Reply:
yes